src/model/fsm_util.jl

```diff
@@ -27,7 +27,7 @@ Returns:
 function load_bvn_mixtures{NumType <: Number}(
```

Comment:
Some of the `Args` listed in the comment are out of date. What about documenting some of the recurring arguments, e.g. `S` and `b` in a comment for the whole `Model` module, rather than in each function that uses them?